project_task_description_template/models/project_project.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    task_template_ids = fields.Many2many(
```

Comment:
Remove please `string` parameter. This parameter is duplicated field name

Reply:
okay, I'll fix it